src/pages/components/DataAndTimeSelector.tsx

```diff
@@ -23,27 +22,34 @@ export function DataAndTimeSelector({
       setTimestamp(timeSelected)
   }, [timeSelected])
 
+  useEffect(() => {
+    if (timeSelected != null && !timeSelected.isSame(timestamp)) setTimeSelected(timestamp)
+  }, [timestamp])
+
+  const width = showTimePicker ? '50%' : '100%'
```

Comment:
You use a built in word as a variable , it is confusing , please change it.

Reply:
change the `timestamp` name to `timeValid`.

Reply:
I meant the 'width' variable name, I should have wrote it precisely

Reply:
OK.
fix in commit [fix space between date and time components](https://github.com/hasadna/open-bus-map-search/pull/61/commits/71f269ef038fe44054d4986684c62ea384e33890) anyway.

the variable not existent anymore.